icinga2-ansible-web2-ui/defaults/main.yml

```diff
@@ -23,3 +23,5 @@ icinga2_web2_ui_rpm:
 
 icinga2_web2_ui_deb:
  - { package: "icingaweb2" }
+ 
+icinga2_timezone: "UTC"
```

Comment:
You assume all users want to use the UTC as default .

Reply:
it would be better to use this fact to determine the individual timezone  icinga2_timezone: "{{ ansible_facts.ansible_date_time.tz  }}"

Reply:
Looking at ansible facts ; ansible_date_time.tz is not is the expected format for PHP timezone ("Europe/Paris" for me).

```
"ansible_date_time": {
            "date": "2018-03-07", 
            [...]
            "tz": "CET", 
            "tz_offset": "+0100", 
            [...]
        }, 
```

Is `/etc/timezone` standard across Unix platforms ?

Reply:
I got my answer here: [How do I find the current system timezone?](https://stackoverflow.com/questions/3118582/how-do-i-find-the-current-system-timezone).

I think it is safe to use `/etc/timezone` as a reference for Debian family OS. Debian has this file since 2010.

Ansible lookups run locally so here is what I came to :